### PR TITLE
EDB Postgres AI - Moving content from BigAnimal to AI docs

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Moved the content from BA to EDB Postgres AI as per the plan in the sheet

### CODE REVIEW

**Comment by josh-heyer** [comment]
FYI, got a link-fixing branch going here: https://github.com/EnterpriseDB/docs/pull/5860
Was able to update a pretty good pile of them programatically, but the rest will need some detective work as *things have moved*

File: advocacy_docs/edb-postgres-ai/cloud-service/references/distributed_high_availability/pgd_cli_ba.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+```bash
+pgd show-nodes --dsn "<your_connection_string>"
+```
+
+## PGD commands in Cloud Service
+
+!!!Note
+
+Three EDB Postgres Distributed CLI commands don't work with distributed high-availability Cloud Service clusters: `create-proxy`, `delete-proxy`, and `alter-proxy-option`. These commands are managed by Cloud Service, as Cloud Service runs on Kubernetes. It's a technical best practice to have the Kubernetes operator handle these functions.
```
Comment (nidhibhammar) [inline advocacy_docs/edb-postgres-ai/cloud-service/references/distributed_high_availability/pgd_cli_ba.mdx:56]:
```suggestion
Three EDB Postgres Distributed CLI commands don't work with distributed high-availability Cloud Service clusters: `create-proxy`, `delete-proxy`, and `alter-proxy-option`. These commands are managed by Cloud Service, as Cloud Service runs on Kubernetes. It's a technical best practice to have the Kubernetes operator handle these functions.
```
```suggestion
Three EDB Postgres Distributed CLI commands don't work with distributed high-availability Cloud Service clusters: `create-proxy`, `delete-proxy`, and `alter-proxy-option`. These commands are managed by Cloud Service, as Cloud Service runs on Kubernetes. Having the Kubernetes operator handle these functions is a technical best 

> practice.

```

File: advocacy_docs/edb-postgres-ai/cloud-service/support_services/index.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Support services
+description: How to create a support case the support portal of Cloud Service.
```
Comment (dbwagoner) [inline advocacy_docs/edb-postgres-ai/cloud-service/support_services/index.mdx:3]:
```suggestion
description: How to create a support case in the support portal of Cloud Service.
```

File: advocacy_docs/edb-postgres-ai/console/using/index.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Using the console
+indexCards: simple
+description: Using the EDB Postgres AI console to from creating clusters and managing users to connecting cloud services and enabling encryption
```
Comment (dbwagoner) [inline advocacy_docs/edb-postgres-ai/console/using/index.mdx:4]: This sentence needs review. For example, "to from".

**Comment by dbwagoner** [comment]
Other than the two comments I left, this looks great to me. I am ready to approve after those two comments are reviewed.

File: advocacy_docs/edb-postgres-ai/console/using/index.mdx (hunk 3 above)
Comment (djw-m) [reply advocacy_docs/edb-postgres-ai/console/using/index.mdx:4]:
```suggestion
description: Using the EDB Postgres AI console from creating clusters and managing users to connecting cloud services and enabling encryption.
```

**Review by djw-m — APPROVED** [review]
LGTM - reviewed content and viewed linkage as much as possible. Solid pass on link checker.

**Pull request merged**